src/main/controllers/multiple_respondent/content.ts

```diff
@@ -0,0 +1,34 @@
+import { YesOrNo } from 'definitions/case';
+import { isFieldFilledIn } from '../../components/form/validator';
+import { FormContent } from '../../definitions/form';
+import { AnyRecord} from '../../definitions/util-types';
+
+export const multipleRespondentContent: FormContent = {
+  fields: {
+    multipleRespondent: {
```

Comment:
Just one small thing, you need to save this value `Case` object so we can later reference it in the session.

Reply:
amended